src/styles/sidebar/components/thread.scss

```diff
@@ -54,6 +54,6 @@
   &__content {
     flex-grow: 1;
     // Prevent annotation content from overflowing the container
-    max-width: 100%;
+    overflow: hidden;
```

Comment:
If we can't find an option but to overflow, I think it would be helpful to provide in the comments an example of the kind of content that might trigger this issue - eg. a long URL with no breakpoints.